cmd/tikv-ctl/src/util.rs

```diff
@@ -62,8 +63,39 @@ pub fn perror_and_exit<E: Error>(prefix: &str, e: E) -> ! {
     tikv_util::logger::exit_process_gracefully(-1);
 }
 
+// Check if the part of region's `key_range` is in the specified
+// `key_range_limit`.
+pub fn check_range_included(key_range: &KeyRange, key_range_limit: &KeyRange) -> bool {
```

Comment:
You mean "intersecetd"?

Reply:
Yes, you've made it clearer. I'll change relevant annotations.